### DISCUSSION TITLE
--image-config-scanners documentation parameter is incorrect

### DISCUSSION THREAD

**Discussion by psg18dhc** [aquasecurity/trivy#6618] [Documentation]
### Description

In the documentation portal it says to use

Container image metadata
Container images have [configuration](https://github.com/opencontainers/image-spec/blob/2fb996805b3734779bf9a3a84dc9a9691ad7efdd/config.md). docker inspect and docker history show the information according to the configuration.

Trivy scans the configuration of container images for

Misconfigurations
Secrets
 >They are disabled by default. You can enable them with --image-config-scanners.

however when i run this option trivy on osx fails. 

Container image metadata
Container images have [configuration](https://github.com/opencontainers/image-spec/blob/2fb996805b3734779bf9a3a84dc9a9691ad7efdd/config.md). docker inspect and docker history show the information according to the configuration.

Trivy scans the configuration of container images for

Misconfigurations
Secrets
They are disabled by default. You can enable them with --image-config-scanners.

### Link

https://aquasecurity.github.io/trivy/v0.50/docs/target/container_image/

### Suggestions

is this incorrect or is there another option to use instead?

**Comment by nikpivkin** [comment]
Hi @psg18dhc !

Track https://github.com/aquasecurity/trivy/issues/6634
